backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/RESTUtil.java

```diff
@@ -148,7 +149,14 @@ public static SyncAdvise mapSyncAdvise(@NonNull final String syncAdvise)
 			case "READ_ONLY":
 				return SyncAdvise.READ_ONLY;
 			default:
-				return null;
+				if (EmptyUtil.isBlank(syncAdvise))
```

Comment:
please avoid such deep nesting by moving this if-block to the start of the method